src/app/shared/pipes/order-by.pipe.spec.ts

```diff
@@ -16,17 +16,13 @@ describe("SharedModule.OrderByPipe:", () => {
 
   it("should transform value if parameter was defined", () => {
     testParameters.forEach(value => {
-      expect(pipe.transform(DEFAULT_CONFIG, value)).toEqual(
-        sortBy(DEFAULT_CONFIG, value),
-      );
+      expect(pipe.transform(DEFAULT_CONFIG, value)).toEqual(sortBy(DEFAULT_CONFIG, value));
     });
   });
 
   it("should transform value if parameter is not defined", () => {
     const sortedTestParameters: Array<string> = testParameters.sort();
-    expect(pipe.transform(DEFAULT_CONFIG)).toEqual(
-      sortBy(DEFAULT_CONFIG, sortedTestParameters),
-    );
+    expect(pipe.transform(DEFAULT_CONFIG)).toEqual(sortBy(DEFAULT_CONFIG, sortedTestParameters));
```

Comment:
better to add "actual" and "expected" variables
and then:
expect(actual).toEqual(expected)